### PR TITLE
TS: using support power can uncloak actor

### PR DESCRIPTION
**Pull request by dnqbob**

### CODE REVIEW

File: OpenRA.Mods.Common/Traits/Cloak.cs [hunk 1]
```diff
@@ -46,8 +47,9 @@ public class CloakInfo : PausableConditionalTraitInfo
 		[Desc("Measured in game ticks.")]
 		public readonly int CloakDelay = 30;
 
-		[Desc("Events leading to the actor getting uncloaked. Possible values are: Attack, Move, Unload, Infiltrate, Demolish, Dock, Damage, Heal and SelfHeal.",
-			"'Dock' is triggered when docking to a refinery or resupplying.")]
+		[Desc("Events leading to the actor getting uncloaked. Possible values are: Attack, Move, Unload, Infiltrate, Demolish, Dock, Damage, Heal, SelfHeal and SupportPower.",
+			"'Dock' is triggered when docking to a refinery or resupplying.",
+			"'SupportPower' is triggered when using support power.")]
```
Comment (abcdefg30) [inline OpenRA.Mods.Common/Traits/Cloak.cs]:
```suggestion
			"'SupportPower' is triggered when using a support power.")]
```

**Review by PunkPun — DISMISSED** [review]
LGTM

**Review by abcdefg30 — DISMISSED** [review]
Lgtm otherwise.

**Review by PunkPun — APPROVED** [review]

File: OpenRA.Mods.Common/Traits/Cloak.cs (hunk 1 above)
Comment (dnqbob) [reply OpenRA.Mods.Common/Traits/Cloak.cs]: fixed

**Comment by PunkPun** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/0076e37822bb0246fcf20fe92ff55bcf16c13577...ef6d8d7345b75f5c61a21b3233327eddd1b86667)

**Pull request merged**